src/eko/msbar_masses.py

```diff
@@ -276,18 +297,8 @@ def compute_msbar_mass(theory_card):
     masses = np.concatenate((np.zeros(nfa_ref - 3), np.full(6 - nfa_ref, np.inf)))
     fact_to_ren = theory_card["fact_to_ren_scale_ratio"] ** 2
 
-    # TODO: why is this different than calling
```

Comment:
For the records: there were two typos:
 * `hqm_scheme="MSBAR"`  was somehow missing
 * `masses`were not divided by `fac_to_ren` which could have led to possible mistakes in the future... 
 Thanks for spotting this @felixhekhorn